pkg/errtypes/errors.go

```diff
@@ -29,8 +29,14 @@ var (
 	// ErrNotImplemented represents that the function is not implemented.
 	ErrNotImplemented = errorType{codeNotImplemented, "not implemented"}
 
-	// ErrUsingbyContainers represents that object is used by containers
+	// ErrUsingbyContainers represents that object is used by containers.
 	ErrUsingbyContainers = errorType{codeInUse, "using by containers"}
+
+	// ErrVolumeInUse represents that volume in use.
+	ErrVolumeInUse = errorType{codeInUse, "volume is in use"}
+
+	// ErrNoVolume represents that no such volume.
+	ErrNoVolume = errorType{codeNotfound, "no such volume"}
```

Comment:
s/ErrNoVolume/ErrVolumeNotFound/g ?

Reply:
make sense

Reply:
@allencloud If I want to add `ErrVolumeNotFound` error type, how to add it?

Reply:
I am afraid at least you should not locate in pkg/errtypes. 
In addition this is not a `code` in errorType. You should not use it in this way.
You just need to construct an error, then just do it, but with no relation errtypes.